src/components/SelectedNote/SelectedNote.jsx

```diff
@@ -0,0 +1,71 @@
+import PropTypes from "prop-types";
+import { Typography } from "@mui/material";
+import CardContent from "@mui/material/CardContent";
+import image from "../../images/main-image.png";
+import IconButton from "@mui/material/IconButton";
+import ModeEditOutlineIcon from "@mui/icons-material/ModeEditOutline";
+import ShareIcon from "@mui/icons-material/Share";
+import DeleteIcon from "@mui/icons-material/Delete";
+import { Box } from "@mui/system";
+
+import {
+  SelectedBox,
+  SelectedCard,
+  SelectedCardActions,
+  SelectedCardContent,
+  SelectedTitle,
+  SelectedDate,
+  MainBox,
+  MainCard,
+  MainCardMedia,
+} from "./styled";
+
+const SelectedNote = ({ note }) => {
+  return (
```

Comment:
There is no need to write `return` statement. We can use arrow function functionality:
```suggestion
const SelectedNote = ({ note }) => (
 <Box>
  ...
```